.travis.yml

```diff
@@ -1,11 +1,11 @@
 language: node_js
 node_js:
   - 6
+  - 7
   - 8
   - 9
+os:
+  - linux
+dist: trusty
```

Comment:
Trusty is the default distro since the 18th of July, 2017 (according to https://docs.travis-ci.com/user/precise-to-trusty-migration-guide/), there is no need to add this line to config.